.github/workflows/unit_tests_cloud.yml

```diff
@@ -157,3 +151,27 @@ jobs:
       run: |
         export PYTHONPATH="$PWD/src:$PYTHONPATH"
         python3 -m unittest src/odemis/acq/test/feature_test.py --verbose
+
+    - name: Run tests from odemis.gui.conf
+      if: success() || failure()
```

Comment:
probably just a personal preference but those `success() || failure()`s in the if statements gives me the feeling that it would run the tests in any case. 

would it be too much of a hassle to change them as `if: '!cancelled()'`?

Reply:
When I implemented everything originally I tried using `if: always()` but that doesn't have the desired result, I now see they added the use `if: '!cancelled()'` to the documentation, I don't think that description was there previously. I'll update the code.
https://docs.github.com/en/actions/learn-github-actions/expressions#always

Reply:
I've updated the code and I followed the suggestion of the github documentation, so I wrote `if: ${{ !cancelled() }}` instead of your suggested `if: '!cancelled()'`. Let me know if you agree or not

Reply:
looks good!